sdk/storagev2_model.go

```diff
@@ -96,9 +98,90 @@ type BcDrivesPage struct {
 }
 
 type BcDrivesPageOptions struct {
+	BcDrivesPageFilters
 	PaginationOrderingOptions
 }
 
+type BcDrivesPageFilters struct {
+	Size            BcDrivesValue64 `url:""`
+	UsedSize        BcDrivesValue64 `url:""`
+	MetaFilesSize   BcDrivesValue64 `url:""`
+	ReplicatorCount BcDrivesValue16 `url:""`
+}
+
+type BcDrivesValue64 struct {
+	Value64           uint64
+	BcDrivesValueType BcDrivesValueType
+}
+
+func (sV BcDrivesValue64) EncodeValues(key string, v *url.Values) error {
```

Comment:
what the point of this solution?

same for `ReplicatorsValue64` and other

Reply:
This code is used for filtering values when retrieving data. The values to filter for BcDrives and Replicators are different. To make it clear, I separated them into their own filter groups. Some filtration values are uint32 or uint16 instead of uint64 and they're further divided into their own groups.

Reply:
I don't see the use of this function. We have the next func that adds parameters to URL. https://github.com/proximax-storage/go-xpx-chain-sdk/blob/385510dbf68f4dcb16af36d35e6492001813205f/sdk/sdk.go#L864 If got you right

Reply:
You're right. I removed them in 4271c12.